### PR TITLE
Enable blacks `skip_magic_trailing_comma` options

### PR DESCRIPTION
**Pull request by headtr1ck**
This little config change will make black remove trailing commas when they are not necessary to fit something into a single line.

It is a pure design choice but personally I like the clean up it does when function signatures simplify (although this happens rarely with more and more type hints added).

I can understand that some people prefer the manual control over what is multiline and what is not.
Feel free to vote on it :)

For me it adds cheap LOCs so it looks like I am working hard, haha.

### CODE REVIEW

File: xarray/core/_aggregations.py [hunk 1]
```diff
@@ -34,11 +34,7 @@ def reduce(
         raise NotImplementedError()
 
     def count(
```
Comment (headtr1ck) [inline xarray/core/_aggregations.py:36]: Note to myself: Fix this in `generate_aggregations` as well...

**Comment by Illviljan** [comment]
I think we should stick to default black values. Otherwise I want to start arguing for line width=79 so we follow pep8. ;)

You can still run this cleanup though so you still get your LOCs. :)

**Comment by dcherian** [comment]
I too vote we stick with black defaults.

**Comment by headtr1ck** [comment]
Ok :)
But I guess it is better to not merge this then, it will only create messy merge conflicts for other PRs.

**Pull request closed without merging**